components/automate-backend-pgleaderchk/habitat/config/config.toml

```diff
@@ -0,0 +1,29 @@
+[database]
+hostname = "{{sys.ip}}"
+{{#if bind.database}}
+  {{#eachAlive bind.database.members as |member|}}
+   {{#if @last}}
+port = {{member.cfg.port}}
+username = "{{member.cfg.superuser_name}}"
```

Comment:
This will suffer from the same issue of open credentials in plain text when the service will be running.